lte/gateway/c/core/oai/test/amf/amf_app_test_util.cpp

```diff
@@ -136,6 +136,18 @@ void send_initial_context_response(
     amf_app_desc_t* amf_app_desc_p, amf_ue_ngap_id_t ue_id) {
   itti_amf_app_initial_context_setup_rsp_t ics_resp = {};
 
+  // apn profile received from subscriberd during location update
+  ue_m5gmm_context_s* ue_context_p =
+      amf_ue_context_exists_amf_ue_ngap_id(ue_id);
+
+  ASSERT_NE(nullptr, ue_context_p);
+
+  apn_config_profile_t& profile = ue_context_p->amf_context.apn_config_profile;
+  profile.nb_apns               = 1;
+  strncpy(
+      profile.apn_configuration[0].service_selection, "internet",
+      SERVICE_SELECTION_MAX_LENGTH - 1);
```

Comment:
Only copy 8 bytes for "internet"
```suggestion
      8);
```

Reply:
comment addressed, parameter replaced with 8